source.go

```diff
@@ -46,6 +46,17 @@ const (
 	SourceMandateNotificationMethodNone   SourceMandateNotificationMethod = "none"
 )
 
+// SourceSourceOrderItemType describes the type of source order items on source
+// orders for sources.
+type SourceSourceOrderItemType string
+
+// The list of possible values for source order item types.
+const (
+	SourceSourceOrderItemTypeSKU      SourceSourceOrderItemType = "sku"
```

Comment:
it looks like you're missing discount. Or the team made discount in the spec but shouldn't have: https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/param/SourceCreateParams.java#L1070